### ISSUE TITLE
Incorrect name when CLI installer is blocked by VS being open

### ISSUE THREAD

**Issue by balachir** [dotnet/cli#2766]
## Steps to reproduce
1. Dev14 U2 RTW + AspNetCore Tools RC2 (build release.106) -- includes DotNetCoreCli.b002426
2. Open ASP.NET Core solution in VS, Ctrl-F5 --> works
3. With VS open, try to install a newer build of DotNetCoreCli : build b002520

## Actual behavior
Installer shows this dialog. The application name that needs to be closed is just "A"

![image](https://cloud.githubusercontent.com/assets/8246794/14926800/e7f55656-0e02-11e6-82b5-a6d5cc1f4733.png)

## Expected  behavior
The application name in the above dialog should be "Visual Studio" ?

**Comment by Sridhar-MS** [comment]
duplicate of #2678

**Comment by Sridhar-MS** [comment]
Workign on a fix - #2764
